src/main/java/com/stripe/model/Account.java

```diff
@@ -116,7 +118,10 @@
   @SerializedName("tos_acceptance")
   TosAcceptance tosAcceptance;
 
-  /** The Stripe account type. Can be `standard`, `express`, or `custom`. */
+  /**
+   * The Stripe account type. Can be <code>standard</code>, <code>express</code>, or <code>custom
+   * </code>.
```

Comment:
Minor but I'm assuming it's okay that this goes to the next line right?

Reply:
It's not perfect because it does cause an extra space to be rendered. I'll see if I can fix it in codegen.

![image](https://user-images.githubusercontent.com/23086931/74204644-e1dd4900-4c28-11ea-9223-7bf8ddf7d26e.png)

Reply:
Looked into it, it's actually a bug with google-java-format: https://github.com/google/google-java-format/issues/84. Unfortunately I don't think we can work around it.

Reply:
We can work around it :) by using `{@code ...}` instead of `<code>...</code>`. This avoids the bug with google-java-format, and is rendered fine by Javadoc:
![image](https://user-images.githubusercontent.com/23086931/74207055-caa35900-4c32-11ea-96bb-e91a11b4abaa.png)